src/main/java/io/axoniq/demo/giftcard/rest/GiftCardController.java

```diff
@@ -0,0 +1,116 @@
+package io.axoniq.demo.giftcard.rest;
+
+import io.axoniq.demo.giftcard.api.command.IssueCardCommand;
+import io.axoniq.demo.giftcard.api.command.RedeemCardCommand;
+import io.axoniq.demo.giftcard.api.query.CardSummary;
+import io.axoniq.demo.giftcard.api.query.CountCardSummariesQuery;
+import io.axoniq.demo.giftcard.api.query.CountCardSummariesResponse;
+import io.axoniq.demo.giftcard.api.query.FetchCardSummariesQuery;
+import org.axonframework.commandhandling.gateway.CommandGateway;
+import org.axonframework.messaging.responsetypes.ResponseTypes;
+import org.axonframework.queryhandling.QueryGateway;
+import org.axonframework.queryhandling.SubscriptionQueryResult;
+import org.springframework.context.annotation.Profile;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.UUID;
+
+@RestController
+@Profile("gui")
+@RequestMapping("/giftcard")
+public class GiftCardController {
+
+    private final CommandGateway commandGateway;
+    private final QueryGateway queryGateway;
+
+    public GiftCardController(CommandGateway commandGateway, QueryGateway queryGateway) {
+        this.commandGateway = commandGateway;
+        this.queryGateway = queryGateway;
+    }
+
+    @GetMapping(path = "/subscribe/limit/{limit}", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<CardSummary> subscribe(
+            @PathVariable int limit
+    ) {
+        return summerySubscription(limit);
+    }
+
+    @GetMapping(path = "/subscribe-count", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<CountCardSummariesResponse> subscribe() {
+        return countSubscription();
+    }
+
+    @PostMapping("issue/id/{id}/amount/{amount}")
+    public Mono<Result> issue(
+            @PathVariable String id,
+            @PathVariable int amount
+    ) {
+        var command = new IssueCardCommand(id, amount);
+        return Mono.fromFuture(commandGateway.send(command))
+                   .map(e -> Result.ok())
```

Comment:
```suggestion
                   .then(Mono.just(Result.ok()))
```
Just to be consistent with the redeem and private issue methods

Reply:
Changed along with the other points.